tests/server/test_calling_routes.py

```diff
@@ -721,6 +721,88 @@ def test_start_recording__returns_error_code_and_message_if_barcode_is_invalid(
     assert response.status.endswith(f"{test_barcode_type} {expected_error_message}") is True
 
 
+@pytest.mark.parametrize("test_barcode_type", ["Plate"])
+@pytest.mark.parametrize(
+    "test_barcode,expected_error_message",
+    [
+        ("M*222123199-1", "barcode is incorrect length"),
+        ("M*222-1", "barcode is incorrect length"),
+        ("MA22123199-1", "barcode contains invalid header: 'MA'"),
+        ("MB22123199-1", "barcode contains invalid header: 'MB'"),
+        ("ME22123199-1", "barcode contains invalid header: 'ME'"),
+        ("M*221$3199-1", "barcode contains invalid char : '221$31991'"),
+        ("M*20123199-1", "year is before 2022: '20'"),
+        ("M*22444199-1", "day is not valid: '444'"),
+        ("M*22123999-1", "experiment id is not valid: '999'"),
+        ("M*22123199-2", "incorrect last digit"),
+    ],
+)
+def test_start_recording__returns_error_code_and_message_if_new_barcode_beta1_mode_scheme_is_invalid(
+    test_barcode_type,
+    test_barcode,
+    expected_error_message,
+    client_and_server_manager_and_shared_values,
+):
+    test_client, _, shared_values_dict = client_and_server_manager_and_shared_values
+    shared_values_dict["beta_2_mode"] = False
+    # dont check stim in beta 1
+    # do last  digit check in its own test
+    # FE refference
+    shared_values_dict["stimulation_running"] = [True] * 24
+
+    barcodes = {
+        "plate_barcode": MantarrayMcSimulator.default_plate_barcode,
+        "stim_barcode": MantarrayMcSimulator.default_stim_barcode,
+    }
+    barcode_type_letter = "L"
+    barcodes[f"{test_barcode_type.lower()}_barcode"] = test_barcode.replace("*", barcode_type_letter)
```

Comment:
Instead of doing this you can just make all the barcodes in the 2nd parametrization have `L` instead of the `*`